content/zh-cn/docs/contribute/style/content-guide.md

```diff
@@ -108,22 +106,18 @@ dual-sourced content.
 Dual-sourced content requires double the effort (or more!) to maintain
 and grows stale more quickly.
 
-{{< note >}}
-
 If you're a maintainer for a Kubernetes project and need help hosting your own docs,
```

Comment:
Please break large chunks into smaller ones.

I think you already know the best practice to translate `{{< note >}}`.

```markdown
{{< note >}}
<!--
English
-->
Translated
{{< /note >}}
```